pyproject.toml

```diff
@@ -78,6 +78,7 @@ dependencies = [
   "Jinja2 < 3.1",
   "tzdata >= 2022.6",
   "pydantic < 2.0",
+  "unidecode <= 1.3.8",
```

Comment:
There is no reason given for this. Why is this added?

Reply:
Sorry, my bad, that should go to a different branch of my fork, something for my own project. Fixing it.